projects/opendr_ws_2/src/opendr_perception/opendr_perception/object_detection_2d_gem_node.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import cv2
+import message_filters
+import numpy as np
+import rclpy
+import torch
+from rclpy.node import Node
+from opendr_ros2_bridge import ROS2Bridge
+from sensor_msgs.msg import Image as ROS_Image
+from vision_msgs.msg import Detection2DArray
+
+from opendr.engine.data import Image
+from opendr.perception.object_detection_2d import GemLearner
+from opendr.perception.object_detection_2d import draw_bounding_boxes
+
+
+class ObjectDetectionGemNode(Node):
+    def __init__(
+        self,
+        input_color_image_topic,
+        input_infra_image_topic,
+        output_color_image_topic,
+        output_infra_image_topic,
```

Comment:
In an earlier commit "color" was changed to "rgb" as far as i can tell, but the changes were reverted, same as in the ROS1 node.